content/_executions.md

```diff
@@ -8,6 +8,7 @@ Attribute | Type | Description
 --------- | ------- | -------
 `blueprint_id` | string | The id of the blueprint the execution is in the context of.
 `created_at` | datetime | The time the execution was queued at.
+`ended_at` | datetime | The time the execution was ended at if in successful, failed or cancelled state.
```

Comment:
\- The time the execution was ended at if in successful, failed or cancelled state.
\+ The time the execution ended in successful, failed or cancelled state.